lib/consoletest.pm

```diff
@@ -51,6 +58,31 @@ sub post_fail_hook {
     $self->export_logs_desktop;
 }
 
+=head2 get_avc_selinux_alerts
+
+List AVCs that have been recorded during a runtime of a test module that executes this function
+
+=cut
+
+sub get_avc_selinux_alerts {
+    if ((current_console() !~ /root|log/) || (script_run('test -f /var/log/audit/audit.log') != 0)) {
+        return undef;
+    }
+
+    my @logged = split(/\n/, script_output('ausearch -m avc -r', proceed_on_failure => 1));
+    $avc_record{end} = scalar @logged - 1;
```

Comment:
If `ausearch` fails or returns less messages than expected, end can go backwards. I suggest moving the assignment past the check below.

Reply:
Ideally this would not skip based on line numbers but timestamps

Reply:
Something like extracting from the last line `msg=audit(1669121672.441:16)` and whatever that comes after ?! Isn't the first float `1669121672.441` a date information? I am thinking whether the output would not get mixed up if the `chrony` would make a time correction somewhere in the middle of the test. On slower machines, we can see here and there that chrony/ntp was doing some time corrections.

Reply:
I'd hope that the logs are still in order, so by omitting everything until the first timestamp match you'll never miss anything new.

Reply:
Ok, let's try it out. I have used array indexes in order to return a slice at the end.